concrete/src/Page/Page.php

```diff
@@ -2599,7 +2664,8 @@ public function updatePermissionsCollectionID($cParentIDString, $npID)
     public function acquireAreaPermissions($permissionsCollectionID)
     {
         $v = [$this->cID];
-        $db = Database::connection();
+        $app = Application::getFacadeApplication();
+        $db = $app->make('database')->connection();
```

Comment:
I hadn't thought about this method ! I used the one described by [the documentation](https://documentation.concrete5.org/developers/appendix/concrete5-version-8-coding-styles/database).
Do you think it's more efficient since it doesn't go through a `$app->bind()` ?

Reply:
It's a really tiny bit faster, but basically it's just a matter of my personal preference: if there's a way to do the same thing with one less call, I opt for it